### PR TITLE
Clarify `getRootNode` return value

### LINKED ISSUE #26962: getRootNode() documentation doesn't mention that it can return the element itself
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/Node/getRootNode

### What specific section or headline is this issue about?

_No response_

### What information was incorrect, unhelpful, or incomplete?

The documentation it is not clear that `getRootNode()` might return the element itself, if it doesn't have a parent.

### What did you expect to see?

Clear statement that the parent of the element matters and also an example.

### Do you have any supporting links, references, or citations?

From [DOM Standard](https://dom.spec.whatwg.org/#dom-node-getrootnode):
> The getRootNode(options) method steps are to return [this](https://webidl.spec.whatwg.org/#this)’s [shadow-including root](https://dom.spec.whatwg.org/#concept-shadow-including-root) if options["[composed](https://dom.spec.whatwg.org/#dom-getrootnodeoptions-composed)"] is true; otherwise [this](https://webidl.spec.whatwg.org/#this)’s [root](https://dom.spec.whatwg.org/#concept-tree-root).

> The root of an object is itself, if its [parent](https://dom.spec.whatwg.org/#concept-tree-parent) is null, or else it is the [root](https://dom.spec.whatwg.org/#concept-tree-root) of its [parent](https://dom.spec.whatwg.org/#concept-tree-parent). The [root](https://dom.spec.whatwg.org/#concept-tree-root) of a [tree](https://dom.spec.whatwg.org/#concept-tree) is any object [participating](https://dom.spec.whatwg.org/#concept-tree-participate) in that [tree](https://dom.spec.whatwg.org/#concept-tree) whose [parent](https://dom.spec.whatwg.org/#concept-tree-parent) is null.

### Do you have anything more you want to share?

`document.createElement("div").getRootNode() // returns the div itself` 

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/node/getrootnode`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Node/getRootNode
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/node/getrootnode/index.md
* Last commit: https://github.com/mdn/content/commit/312081aabba3885b35a81107b3c2fc53428896c5
* Document last modified: 2023-04-07T06:19:30.000Z

</details>

### PR DESCRIPTION
**Pull request by arminv**
### Description

Make it clear that if an element has no parent, calling `getRootNode` on it will return the element itself. Also provide a simple example for this case.

### Motivation

Make it more clear to readers.

### Additional details

None.

### Related issues and pull requests

Fixes #26962

### CODE REVIEW

File: files/en-us/web/api/node/getrootnode/index.md [hunk 1]
```diff
@@ -91,6 +97,15 @@ output.textContent += `shadowChild.getRootNode({composed:true}).nodeName : ${
 
 {{ EmbedLiveSample('Example 2', '100%', '200px') }}
 
+### Example 3
+
+This example returns the element itself as it has no parent:
+
+```js
+const element = document.createElement("p");
+const rootNode = element.getRootNode(); // #p
```
Comment (Josh-Cena) [inline files/en-us/web/api/node/getrootnode/index.md:106]: What about a slightly more complicated one?

```js
const element = document.createElement("p");
const child = document.createElement("span");
element.append(child);
const rootNode = child.getRootNode(); // <p><span></span></p>
```

It returns the root of the unmounted tree, not the element itself.

File: files/en-us/web/api/node/getrootnode/index.md [hunk 2]
```diff
@@ -32,12 +32,18 @@ getRootNode(options)
...
-- Calling it on an element inside a shadow DOM will return the associated
-  {{domxref("ShadowRoot")}}.
+- where you call `getRootNode()`; for example:
+
+  - Calling it on an element inside a standard web page will return an
+    {{domxref("HTMLDocument")}} object representing the entire page (or {{HTMLElement("iframe")}}).
+  - Calling it on an element inside a shadow DOM will return the associated
+    {{domxref("ShadowRoot")}}.
+
+- whether an element has a parent or not:
```
[the remark this answers is not in the document]
Comment (arminv) [reply files/en-us/web/api/node/getrootnode/index.md:44]: I see your point thanks for clarifying this!

File: files/en-us/web/api/node/getrootnode/index.md (hunk 1 above)
Comment (arminv) [reply files/en-us/web/api/node/getrootnode/index.md:106]: Yeah your example is more in line with the updated point above. I just wonder if there is any value in adding equality checks to the example code, and a note to its description? My understanding was the original issue asked for more clarity around this specifically.

- "This example returns the root of the unmounted tree. Note `element` here is the root of the tree (as it has no parent), so by definition its root is itself."

```js
const element = document.createElement("p");
const child = document.createElement("span");

element.append(child);

const rootNode = child.getRootNode(); // <p><span></span></p>

element === rootNode; // true
element === element.getRootNode(); // true
```

Comment (Josh-Cena) [reply files/en-us/web/api/node/getrootnode/index.md:106]: @arminv This proposed example LGTM. You can make any edits you want and then we can talk based on the updated change.

Comment (arminv) [reply files/en-us/web/api/node/getrootnode/index.md]: @Josh-Cena Thanks for checking this! I have addressed both points now.

**Review by Josh-Cena — APPROVED** [review]
This looks great. Thanks for working on this!

**Pull request merged**